docs/FAQ/identity_management_FAQ.md

```diff
@@ -1,18 +1,18 @@
 ## Identity Management (Membership Service)
 &nbsp;
 ##### What is unique about the fabric's Membership Service module?
-The design and implementation of the Membership Service module encompass many of the latest advances in cryptography, which we believe make it stand apart from other alternatives.
+One of the things that makes the Membership Service module stand out from the pack is our implementation of the latest advances in cryptography.
```

Comment:
this is a bit bold... sounds like marketing material. Shouldn't we back this claim up with some evidence?